### ISSUE TITLE
Unable to do multi-insert according to docs

### ISSUE THREAD

**Issue by housleyjk** [aio-libs/aiopg#46]
The following code should work according to http://aiopg.readthedocs.org/en/stable/sa.html#aiopg.sa.SAConnection.execute:
```python
import asyncio
from aiopg.sa import create_engine
import sqlalchemy as sa


metadata = sa.MetaData()

table = sa.Table(
    'table',
    metadata,
    sa.Column('id', sa.Integer, primary_key=True),
    sa.Column('value', sa.String(255)),
)


@asyncio.coroutine
def go():
    engine = yield from create_engine('postgresql://localhost/aiopg_issue')

    with (yield from engine) as conn:
        yield from conn.execute(
            table.insert(),
            {"id": 1, "value": "v1"},
            {"id": 2, "value": "v2"},
        )

        res = yield from conn.execute(table.select())
        for row in res:
            print(row.id, row.val)


loop = asyncio.get_event_loop()
loop.run_until_complete(go())
```

However, I get a `aiopg.sa.exc.ArgumentError: Don't mix sqlalchemy clause and execution with parameters` error because of the dictionaries passed to execute. Are the docs out of date?

**Comment by asvetlov** [comment]
Yes, I knew the issue.
You are the first person who asked for fix.
I'll make new release with bugfix very soon, thanks.

**Comment by asvetlov** [comment]
Multi-insert are disabled (psycopg2 doesn't support the feature in async mode), docs are cleaned up to don't mention the feature.

`SAConnection.execute` now accepts dict, list, tuple, named and positional params for specifying row data in INSERT SQL statement.

Released in 0.6.1

**Comment by housleyjk** [comment]
Thank you! That clears things up.
